samcli/lib/build/app_builder.py

```diff
@@ -771,7 +771,7 @@ def _get_build_options(
                 base_dir, metadata, source_code_path, scratch_dir
             )
             if working_directory:
-                options = {**options, "working_directory": working_directory}
+                options = {**options, "working_directory": working_directory.replace("\\", "/")}
```

Comment:
Should we instead use pathlib here?

Reply:
not sure, will it do the same thing ? .. I want to pass the path in a unix format, so it can work in windows, and also can work if customer run `sam build --use-container` from a windows machine